fdbserver/KeyValueStoreShardedRocksDB.actor.cpp

```diff
@@ -224,7 +224,7 @@ Error statusToError(const rocksdb::Status& s) {
 	if (s.IsIOError()) {
 		return io_error();
 	} else if (s.IsTimedOut()) {
-		return transaction_too_old();
+		return key_value_store_deadline_exceeded();
```

Comment:
How does SS handle this error? Will EagerReads be retried?

Reply:
No. As discussed, if this error is encountered, we could adjust the timeout accordingly.